client/main.lua

```diff
@@ -170,18 +169,20 @@ RegisterNUICallback('volumeDown', function(_, cb)
 end)
 
 RegisterNUICallback('increaseradiochannel', function(_, cb)
-    local newChannel = radioChannel + 1
-    exports['pma-voice']:setRadioChannel(newChannel)
-    exports.qbx_core:Notify(Lang:t('new_channel')..newChannel, 'success')
+    if not onRadio then return end
+    radioChannel += 1
+    exports['pma-voice']:setRadioChannel(radioChannel)
+    exports.qbx_core:Notify(Lang:t('new_channel')..radioChannel, 'success')
     cb('ok')
 end)
 
 RegisterNUICallback('decreaseradiochannel', function(_, cb)
     if not onRadio then return end
-    local newChannel = radioChannel - 1
-    if newChannel >= 1 then
-        exports['pma-voice']:setRadioChannel(newChannel)
-        exports.qbx_core:Notify(Lang:t('new_channel')..newChannel, 'success')
+    radioChannel -= 1
+    radioChannel = radioChannel < 1 and 1 or radioChannel
+    if radioChannel >= 1 then
```

Comment:
This check is also redundant due to the ternary operator above making sure it doesn't go below 1

Reply:
already fixed :)